addon/ng2/blueprints/ng2/files/karma.conf.js

```diff
@@ -0,0 +1,25 @@
+module.exports = function(config) {
+  config.set({
+    basePath: '',
+    frameworks: ['jasmine'],
+    files: [
+      {pattern: 'node_modules/systemjs/dist/system.src.js', included: true, watched: true},
+      {pattern: 'node_modules/angular2/bundles/angular2.js', included: true, watched: true},
+      {pattern: 'node_modules/angular2/bundles/testing.js', included: true, watched: true},
+      
+      {pattern: 'karma-test-shim.js', included: true, watched: true},
+      {pattern: 'src/test/matchers.js', included: true, watched: true},
+      
+      {pattern: 'dist/**/*.js', included: false, watched: true}
+    ],
```

Comment:
also I don't see the `proxy` configuration to enable component template fetching from tests